publish.sh

```diff
@@ -64,6 +64,15 @@ $(pwd)/runs/${run_id}"
 spark_config=runs/$run_id/regular.spark.properties
 cat conf/spark.properties.template /usr/lib/spark2/conf/spark-defaults.conf > ${spark_config}
 
+# Generate placeholder_images parquet file
+echo "Generating placeholder_image parquet file"
+STARTTIME=${SECONDS}
+python placeholder_images.py ${monthly_snapshot}
```

Comment:
If you alter SparkSession init, the job should be submitted with `spark2-submit` instead of `python ...`.